apps/chat/src/components/Chat/ShareModal.tsx

```diff
@@ -79,7 +79,7 @@ export default function ShareModal() {
   return (
     <Modal
       portalId="theme-main"
-      containerClassName="inline-block w-full max-w-[424px] p-6"
+      containerClassName="inline-block w-full max-w-[424px] px-3 py-4 md:p-6"
```

Comment:
Can we move `px-3 py-4 md:p-6` in base component `Modal` instead?

Reply:
I removed individual styles from other places and added this code to the modal in a new commit